forte/api/forte_python_module.cc

```diff
@@ -162,9 +162,18 @@ PYBIND11_MODULE(forte, m) {
     m.def("make_fragment_projector", &make_fragment_projector,
           "Make a fragment(embedding) projector");
     m.def("make_embedding", &make_embedding, "Apply fragment projector to embed");
-    m.def("make_ints_from_psi4", &make_forte_integrals_from_psi4,
-          "Make Forte integral object from psi4");
-    m.def("make_custom_ints", &make_custom_forte_integrals, "Make a custom integral object");
+    m.def("make_custom_ints", &make_custom_forte_integrals,
+          "Make a custom Forte integral object from arrays");
+    m.def("make_ints_from_psi4",
```

Comment:
Why do we have two separate functions for this?

If we want to retire implicitly grabbing `SCF_TYPE` from options, let's add a deprecation warning. If we don't want to retire it, why not combine these two functions into a single function with an optional `std::string` argument?

Reply:
I combined the two functions. Eventually we want to deprecate, but that's for another time.